pkg/frontend/adminactions/azureactions.go

```diff
@@ -30,11 +30,12 @@ type azureActions struct {
 	env env.Interface
 	oc  *api.OpenShiftCluster
 
-	resources       features.ResourcesClient
-	virtualMachines compute.VirtualMachinesClient
-	virtualNetworks network.VirtualNetworksClient
-	routeTables     network.RouteTablesClient
-	storageAccounts storage.AccountsClient
+	resources         features.ResourcesClient
+	virtualMachines   compute.VirtualMachinesClient
+	virtualNetworks   network.VirtualNetworksClient
+	diskEncryptionSet compute.DiskEncryptionSetsClient
```

Comment:
nit: we seem to use plural form for rest of the clients, but not for tihs one.

Reply:
Changed.